src/lib/ws/server.ts

```diff
@@ -14,8 +15,10 @@ export type WsServer = Server<DefaultEventsMap, DefaultEventsMap, DefaultEventsM
 
 export const PROBES_NAMESPACE = '/probes';
 const TIME_UNTIL_VM_BECOMES_HEALTHY = 8000;
+const TIME_TO_CACHE_FETCH_SOCKETS = 200;
 
 let io: WsServer;
+let throttledFetchSockets: _.DebouncedFunc<() => Promise<Array<RemoteSocket<DefaultEventsMap, SocketData>>>>;
```

Comment:
You're mixing up words throttle and debounce, which have different meanings (https://css-tricks.com/debouncing-throttling-explained-examples/)

Reply:
`_.throttle` itself returns `_.DebouncedFunc`, seems they are using the same type for both methods.
<img width="859" alt="image" src="https://user-images.githubusercontent.com/16905445/206186908-a40e477b-cdfa-4541-b0df-b87fe699c3a8.png">